boa/interpret.py

```diff
@@ -129,7 +129,7 @@ def compiler_data(
     global _disk_cache, _search_path
 
     file_input = FileInput(
-        source_code=source_code,
```

Comment:
i think this change was already made in https://github.com/vyperlang/titanoboa/pull/227 right?

Reply:
yes but this is #220 so it was created before. Next push will include those commits so this should disappear